crawl-ref/source/ghost.cc

```diff
@@ -159,18 +159,21 @@ void ghost_demon::reset()
  */
 static brand_type _random_special_pan_lord_brand()
 {
-    return random_choose(SPWPN_FLAMING,
-                         SPWPN_FREEZING,
-                         SPWPN_ELECTROCUTION,
-                         SPWPN_VENOM,
-                         SPWPN_DRAINING,
-                         SPWPN_SPEED,
-                         SPWPN_VORPAL,
-                         SPWPN_VAMPIRISM,
-                         SPWPN_PAIN,
-                         SPWPN_ANTIMAGIC,
-                         SPWPN_DISTORTION,
-                         SPWPN_CHAOS);
+    return random_choose_weighted(10, SPWPN_FLAMING,
+                                  10, SPWPN_FREEZING,
+                                  10, SPWPN_ELECTROCUTION,
+                                  10, SPWPN_VENOM,
+                                  10, SPWPN_SPEED,
+                                  // Lower chance
+                                  5, SPWPN_DRAINING,
+                                  5, SPWPN_VORPAL,
```

Comment:
I'm thinking maybe not have vorpal at all, since as far as the player can tell it's ~indistinguishable from having no brand.